### PR TITLE
Replaced foreign keys with simple indexes

### LINKED ISSUE #97: [Planetscale Compatibility] Add option to skip foreign key constraints
I'm working on testing Statamic with PlanetScale and ran into this issue here:

```
 SQLSTATE[HY000]: General error: 1105 foreign key constraints are not allowed, see https://vitess.io/blog/2021-06-15-online-ddl-why-no-fk/ (SQL: alter table `form_submissions` add constraint `form_submissions_form_id_foreign` foreign key (`form_id`) references `forms` (`id`) on delete cascade)
 ```

I'm not quite sure what the best way to check for configure excluding [this line](https://github.com/statamic/eloquent-driver/blob/master/database/migrations/create_form_submissions_table.php.stub#L12). 

Thoughts?

### PR DESCRIPTION
**Pull request by FrittenKeeZ**
Also added index to entries order, and re-aligned migrations with Laravel's stub.

Fixes #97

### CODE REVIEW

File: database/migrations/create_entries_table_with_string_ids.php.stub [hunk 1]
```diff
@@ -10,22 +10,18 @@ return new class extends Migration {
         Schema::create($this->prefix('entries'), function (Blueprint $table) {
             $table->uuid('id');
             $table->string('site')->index();
-            $table->uuid('origin_id')->nullable();
```
Comment (jasonvarga) [inline database/migrations/create_entries_table_with_string_ids.php.stub:13]: Was thing change from `uuid` to `unsignedBigInteger` intentional? The origin id would also be a uuid.

**Comment by jasonvarga** [comment]
Sorry I deleted my comment because right after I realized how long this has been opened, I didn't want to bother you. I was going to just sort it out myself. 😊 But thanks for updating that.

**Comment by ryanmitchell** [comment]
@jasonvarga I know you've got a seperate fix for the FK errors at https://github.com/statamic/eloquent-driver/pull/160 but I still think this is worth merging as it solves Jack's original issue. Do you have any concerns?

**Comment by ryanmitchell** [comment]
@FrittenKeeZ I've added an update script to provide migrations for existing installs so that the table definitions match going forward.

**Pull request merged**